drivers/docker/driver.go

```diff
@@ -1324,6 +1325,9 @@ func (d *Driver) createContainerConfig(task *drivers.TaskConfig, driverConfig *T
 		if glob.Glob(configurationExtraLabel, "job_id") {
 			labels[dockerLabelJobID] = task.JobID
 		}
+		if glob.Glob(configurationExtraLabel, "parent_job_id") {
+			labels[dockerLabelParentJobID] = task.ParentJobId
+		}
```

Comment:
Most jobs won't have a parent job ID (the value will be `""`), only dispatch and periodic jobs do. So we probably don't want to add the label if it's not set. That'll also change the test case you have below.

Reply:
@tgross I understand that the parent job id will be empty if the job is not dispatch or periodic. So should we check for an extra condition if the  **_task.ParentJobId_** is empty string, then we skip the label?

Reply:
Right!